docs/user/aws/limits.md

```diff
@@ -5,29 +5,88 @@ You can find a comprehensive list of the default AWS service limits published he
 [AWS Service Limits][service-limits]
 
 Below, we'll identify OpenShift cluster needs and how those impact some of those limits.
+We will also use [aws cli](https://aws.amazon.com/cli/) and [jq](https://stedolan.github.io/jq/) to try and explan
+where limits are set (viewable) and how to show your current usage of resources.
+
+Many commands below are based on [Trusted Advisor](https://aws.amazon.com/premiumsupport/technology/trusted-advisor/)
+and often this can be used to print both the limit and the usage for a service. However to check those limits/usage
+you need to first identify a check_id. These check_id's can be pulled from:
+
+```sh
+$ aws support describe-trusted-advisor-checks --language en | jq -c '.checks[] | {ID: (.id), Name: .name, Category: .category}'
+```
+
+Should you need to increase a limit (with any of the following) you need to open a ticket with AWS support, according to
+[AWS Service Limits](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html).
+
+### Example: Using N. Virginia (us-east-1) to increase a limit for a VPC Elastic IP
+
+To use N. Virginia (us-east-1) for a new cluster, please submit a limit increase for VPC Elastic IPs similar to the
+following in the support dashboard (to create more than one cluster, a higher limit will be necessary):
+
+![Increase Elastic IP limit in AWS](images/support_increase_elastic_ip.png)
 
 ## S3
 
 There is a default limit of 100 S3 buckets per account. The installation creates a bucket temporarily. Also, the
 registry component creates a permanent bucket. This will limit the number of clusters per account to 99 initially. To
 support additional clusters, you must open a support case with AWS.
 
+You can not view the limits for S3 buckets, however you can view how many buckets you are using (what the name and
+creation date is), by running the following:
+
+```sh
+$ aws --region us-east-1 s3api list-buckets | jq -c '.Buckets[]'
+```
+
 ## VPC
 
 Each cluster creates its own VPC. The default limit of VPCs per region is 5 and will allow 5 clusters. To have more
 than 5 clusters, you will need to increase this limit.
 
-Each cluster also creates a VPC Gateway Endpoint for a stable connection to S3. The default limit of VPC Gateway 
-Endpoints per region is 20 and will allow 20 clusters. To have more than 20 clusters, you will need to increase this 
+To see both the limit and the usage for VPC's you can use:
+
+```sh
+$ aws support describe-trusted-advisor-check-result --check-id jL7PP0l7J9 | jq -c '.result.flaggedResources[] | {service_limit: (.metadata[2]), status: (.status), region: (.region), limit: (.metadata[3]), usage: (.metadata[4])}'
+```
+
+You can confim the usage in a region by running the following:
+
+- Be sure to change the region (from us-east-1) if you are using a different region.
+
+```sh
+$ aws --region us-east-1 ec2 describe-vpcs | jq -c '.Vpcs[] | {vpcID: .VpcId, State: .State}'
```

Comment:
Do we care about this?  Confirming a count returned by `describe-trusted-advisor-check-result` doesn't seem like the strongest motivation.